pages/electronics-library.tsx

```diff
@@ -296,24 +320,25 @@ const Home = ({
 
       <div className={styles.electronicsContainer}>
         {filteredComponents.map((item: any) => (
-          <div
-            key={item.name + item.shortDescription.substring(0, 30)}
-            className={styles.electronicsItem}
-            onClick={() => {
-              setShowOverlay(true);
-              setActiveProduct(item);
-            }}
-          >
-            <div className={styles.productName}>{item.name}</div>
-            <img src={item.productPhoto[0]} alt={item.name} />
+          <Link href={`/electronics-library#${item.id}`} key={item.id + item.shortDescription.substring(0, 30)}>
             <div
-              className={styles.category}
-              style={styleFromCategory(item.category)}
+              className={styles.electronicsItem}
+              onClick={() => {
+                setShowOverlay(true);
+                setActiveProduct(item);
+              }}
```

Comment:
Is this still necessary given that the link would handle this for us?

Reply:
Good catch!